pycqed/instrument_drivers/physical_instruments/Fridge_monitor.py

```diff
@@ -112,13 +171,16 @@ def _update_monitor(self):
         time_since_update = time() - self._last_temp_update
         if time_since_update > (self.update_interval()):
             self.temp_dict = {}
-            try:
+            self.press_dict = {}
+            # try:
+            if 1:
```

Comment:
@brianzi I see you have replaced the try/except statement with an always True mode. The try/except is there for the experiments to keep running when the fridge mon website gives an error for some reason. Could you explain why you changed this? (I assume it was for debugging). 

Maybe we should add a comment as to why the try/except is there.